apps/science/CMakeLists.txt

```diff
@@ -1,4 +1,16 @@
 add_executable(science.${TARGET}-board.elf)
```

Comment:
I've committed a PR which moved around some of the Encoders around the repo. This shouldn't affect the functionality of this code but you may need to edit your makefiles to find the files once that is merged.

Reply:
Ah sounds good thanks for the heads up, Ill keep an eye out for that.